beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/EntityHousekeepingPath.java

```diff
@@ -35,7 +35,7 @@
 
 @Entity
 @Table(name = "path")
-public class EntityHousekeepingPath implements HousekeepingPath {
+public class EntityHousekeepingPath implements Housekeeping {
```

Comment:
Could we now rename this to `HousekeepingPath` since the interface now has a different name? That might also require a table name change?